### PR TITLE
XIVY-12602 Integrate inscription view into process editor 

### PR DESCRIPTION
**Pull request by ivy-lli**
- Remove inscription webview
- Add inscription webview abilities to process editor
- Change process editor webview build to vite

### CODE REVIEW

File: webviews/process-editor/src/app.ts [hunk 1]
```diff
@@ -47,5 +62,20 @@ class IvyGLSPStarter extends GLSPStarter {
...
 
 export function launch(): void {
+  MonacoUtil.initStandalone(editorWorker);
+  MonacoEditorUtil.initMonaco(reactMonaco, 'light');
+  const handleMessages = (event: MessageEvent<Message>) => {
+    const message = event.data;
+    switch (message.command) {
+      // case 'connect.to.web.sockets':
+      //   handleConnectToWebSocketsCommand(message);
+      //   break;
```
Comment (ivy-lli) [inline webviews/process-editor/src/app.ts:72]: remove

**Review by ivy-lli — COMMENTED** [review]
Fix integration tests

**Review by ivy-cst — APPROVED** [review]
:trophy:

**Review by ivy-lmu — COMMENTED** [review]
👍

File: webviews/process-editor/vite.config.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig(() => {
+  const config = {
+    plugins: [tsconfigPaths()],
+    build: {
+      manifest: 'build.manifest.json',
+      outDir: '../dist/process-editor',
+      chunkSizeWarningLimit: 5000
+      // rollupOptions: {
```
[the remark this answers is not in the document]
Comment (ivy-lli) [reply webviews/process-editor/vite.config.js:12]: not yet, but maybe later :)

File: package.json [hunk 3]
```diff
@@ -21,8 +21,7 @@
     "build:node": "webpack --mode=development",
     "build:production": "lerna run build:production && webpack --mode=production",
     "download:engine": "ts-node scripts/download-engine.ts",
-    "clean": "lerna run clean & yarn clean:node & rimraf webviews/dist",
-    "clean:node": "rimraf lib dist",
```
[the remark this answers is not in the document]
Comment (ivy-lli) [reply package.json:25]: Thanks for the hint. will fix it in the next pr

**Pull request merged**